src/components/elements/DidSearch.tsx

```diff
@@ -118,7 +118,6 @@ export const DidSearch: React.FunctionComponent<DidSearchProps> = (props) => {
                       }}
                       did={{
                         did: did.did,
-                        // @TODO/tobek Is this how we'll detect messaging support for now?
                         messagingSupported: !!did.services?.length,
```

Comment:
so this *is* how we're detecting messaging support? shouldn't it check if services contains a DIDComm service?

Reply:
@nickreynolds I didn't mean to delete that comment. Currently there's a bug where we're stringing DIDs together and if any of the DIDs has a service endpoint we say that they all do. This just updates it to check the diddoc so we're checking on each DID. Is there a specific service type I should check instead?